.sauce/config.yml

```diff
@@ -23,18 +23,18 @@ suites:
   - name: "Chrome using global mode setting" # Since the suite doesn't specify the `mode`, it'll inherit the mode specified via `defaults.mode` (see line number 3 and 4 of this config file).
     browserName: "chrome"
     src:
-      - "tests/*.test.js" # test files glob
+      - "tests/example.test.js" # test files glob
```

Comment:
Hm. Why? 🤔 
What's so special about `example.test.js` vs. `example.ios.test.js` where one of them can only run on ios and the other one not?

I also don't think we quite a need a separate config for this. It's not like it's an entirely different setup. It's suggesting to the user that they need a separate config file entirely to be able to run their testcafe tests against iOS devices, but that shouldn't be the case.
I'd recommend to just add a suite to the existing config that utilizes ios 💪

Reply:
As we agreed I added more tests and put the iOS suite to a config.yml